### ISSUE TITLE
Save model to disk

### ISSUE THREAD

**Issue by code-mc** [lyst/lightfm#132]
I checked the documentation and couldn't find any info on this. Is it possible to serialize a model to disk? Would be useful when training takes multiple hours.

**Comment by maciejkula** [comment]
You can pickle it like any sklearn model: https://github.com/lyst/lightfm/blob/master/tests/test_movielens.py#L437

**Comment by code-mc** [comment]
Ok thanks!

**Comment by stuartcrobinson** [comment]
i think this line number might be out-dated:

https://github.com/lyst/lightfm/blob/master/tests/test_movielens.py#L437

**Comment by maciejkula** [comment]
But the advice on pickling stands: https://github.com/lyst/lightfm/blob/master/tests/test_movielens.py#L476

**Comment by ratikapoor** [comment]
When I am saving the model on disk using pickle and again loading it, the accuracy metrics are changing on the test set. Should I not use pickle to save the model?
